core/commands/add.go

```diff
@@ -327,6 +327,14 @@ You can now check what blocks have been created by:
 						return
 					}
 				}
+
+				if bar.Total == 0 && bar.Get() != 0 {
```

Comment:
We should only do this if `progress` is true.

Reply:
Thx a lot. I have made changes to this.